node/runtime/src/lib.rs

```diff
@@ -83,16 +69,17 @@ use constants::{time::*, currency::*};
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-/// This runtime version.
+/// Runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("edgeware"),
+	spec_name: create_runtime_str!("node"),
```

Comment:
I believe the spec name should be `edgeware`.

Reply:
will change.